internal/web/tool.go

```
@@ -191,60 +295,129 @@ func runTool(vaultsDataFile []ui.VaultsDataFile, vaultID *string, nonceOverride,
-			if _, ok := vaultAllSharesECDSA[vID]; !ok {
-				vaultAllSharesECDSA[vID] = make([]*ecdsa_keygen.LocalPartySaveData, 0, len(sharesECDSA))
-			}
-			vaultAllSharesECDSA[vID] = append(vaultAllSharesECDSA[vID], vaultSharesECDSA...)
-			// / ECDSA
-			// EDDSA
-			if sharesEDDSA != nil {
-				if vaultSharesEDDSA, welp = inflateSharesForCurve[eddsa_keygen.LocalPartySaveData](sharesEDDSA, justListingVaults); welp != nil {
+				// Build up shares lists
+				// - Ensure that ECDSA shares were found.
+				// - EdDSA shares may not be set for a legacy vault, so we won't catch that as a blocking issue
+				vaultSharesECDSA, vaultSharesEDDSA := make([]*ecdsa_keygen.LocalPartySaveData, 0), make([]*eddsa_keygen.LocalPartySaveData, 0)
```

golangci-lint (2.12.2)

[error] ineffectual assignment to vaultSharesECDSA

(ineffassign)